packages/cashier/src/app.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { setWebsocket } from '@deriv/shared';
 import { init } from 'Utils/server_time';
-import Routes from 'Containers/routes';
 import CashierProviders from './cashier-providers';
+import AppContent from './app-content';
 
 const App = ({ passthrough: { WS, root_store } }) => {
     const { notification_messages_ui: Notifications } = root_store.ui;
```

Comment:
@heorhi-deriv Let's do this in the `AppContent` component and access it via the `useStore` hook instead of directly using the prop.